core/src/components/input-password-toggle/input-password-toggle.tsx

```diff
@@ -1,11 +1,13 @@
+import eyeSlashRegular from '@phosphor-icons/core/assets/regular/eye-slash.svg';
+import eyeRegular from '@phosphor-icons/core/assets/regular/eye.svg';
```

Comment:
Originally I was importing the icons without the fill word `Regular`, but this component in particular already has the `eye` import here:

```tsx
import { eyeOff, eye } from 'ionicons/icons';
```

In order to avoid naming conflicts I appended the fill type to all icon imports. Additionally, if we ever want to use a different fill of the same icon for a different theme, this approach is more future-proof.